src/app/modules/form/admin-form/admin-form.utils.ts

```diff
@@ -169,6 +170,11 @@ export const mapRouteError = (
         statusCode: StatusCodes.UNPROCESSABLE_ENTITY,
         errorMessage: error.message,
       }
+    case PaymentChannelNotFoundError:
+      return {
+        statusCode: StatusCodes.BAD_REQUEST,
```

Comment:
```suggestion
        statusCode: StatusCodes.FORBIDDEN,
```

I don't think it should be a general bad request error code as that is too generic. I think 403 Forbidden might be more suitable, especially based on [Mozilla's documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403).
For consideration: [List of HTTP status codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)

Reply:
Thanks for correcting me on this, was unsure which code to return! [6aa8f60](https://github.com/opengovsg/FormSG/pull/6258/commits/6aa8f6026f7873f170a2b9c0f2a9fbb6adaccef9)